app/models/collection.rb

```diff
@@ -6,20 +6,20 @@ class Collection < ActiveRecord::Base
 
   has_and_belongs_to_many :projects
   has_many :collections_subjects,
-    -> { order(id: :desc) },
-    dependent: :destroy
+           -> { order(id: :desc) },
+           dependent: :destroy
   has_many :subjects, through: :collections_subjects
-  has_many :collection_roles, -> { where.not("access_control_lists.roles = '{}'") }, class_name: "AccessControlList", as: :resource
+  has_many :collection_roles, -> { where.not("access_control_lists.roles = '{}'") }, class_name: 'AccessControlList', as: :resource
   has_many :user_collection_preferences, dependent: :destroy
-  belongs_to :default_subject, :class_name => "Subject"
+  belongs_to :default_subject, class_name: 'Subject'
 
   validates :display_name, presence: true
   validates :private, inclusion: { in: [true, false], message: "can't be blank" }
-  ## TODO: This potential has locking issues
-  validates_with UniqueForOwnerValidator
+  ## TODO: UniqueForOwnerValidator potential has locking issues
+  validates_with UniqueForOwnerValidator, Validators::OneFavoritePerOwnerValidator
```

Comment:
not blocking, perhaps move / refactor existing validator `UniqueForOwnerValidator` code to align with your new validator organization folder structure? for consistency in the application going forward?

Reply:
Sure thing, that sounds sensible